### PR TITLE
Add convergence (integration) test for MultitaskClassifier

### PR DESCRIPTION
**Pull request by bhancock8**
## Description of proposed changes
Add integration test confirming that MultitaskClassifier with two tasks (with simple, linear decision boundaries) converges to near-perfect scores when trained.

## Test plan
Added new test.

## Checklist
* [x] I have read the **CONTRIBUTING** document.
* [x] I have updated the documentation accordingly.
* [x] I have added tests to cover my changes.
* [x] All new and existing tests passed.

### CODE REVIEW

File: test/classification/test_classifier_convergence.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            val_loss_output = model.calculate_loss(
+                val_dataset.X_dict, val_dataset.Y_dict
+            )
+            val_loss = val_loss_output[0][task_name].item()
+            self.assertLess(val_loss, 0.05)
+
+
+def create_data(n: int, offset=0) -> pd.DataFrame:
+    X = np.random.random((n, 2)) * 2 - 1
+    Y = (X[:, 0] < X[:, 1] + offset).astype(int)
```
Comment (vincentschen) [inline test/classification/test_classifier_convergence.py]: brief comment explaining what's going on with the X indexes?
